pkg/utils/options/options.go

```diff
@@ -37,12 +37,13 @@ const (
 type Options struct {
 	*flag.FlagSet
 	// Vendor Neutral
-	MetricsPort     int
-	HealthProbePort int
-	KubeClientQPS   int
-	KubeClientBurst int
-	EnableProfiling bool
-	MemoryLimit     int64
+	MetricsPort           int
+	HealthProbePort       int
+	KubeClientQPS         int
+	KubeClientBurst       int
+	EnableProfiling       bool
+	DisableLeaderElection bool
```

Comment:
Is there a reason this is `DisableLeaderElection` rather than just `LeaderElect`.  kube-scheduler uses `--leader-elect`. It's awkward too since we have `EnableProfiling`.  I'm assuming this was to make the default value the primitive default initialized value but I'm not sure it makes sense in this case. 

https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/#:~:text=%2D%2Dleader%2Delect%C2%A0%C2%A0%C2%A0%C2%A0%C2%A0Default%3A%20true

https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/#:~:text=%2D%2Dleader%2Delect%C2%A0%C2%A0%C2%A0%C2%A0%C2%A0Default%3A%20true

Reply:
Good catch. Yeah, let's keep parity with master node controllers.